uportal-war/src/main/webapp/WEB-INF/flows/edit-portlet/choosePortlet.jsp

```diff
@@ -87,7 +87,7 @@ PORTLET DEVELOPMENT STANDARDS AND GUIDELINES
             </div>
             <!-- Portlet Dropdown -->
             <div class="form-group">
-                <div class="col-md-3">
+                <div class="col-md-3"><spring:message code="portlet.information"/>
```

Comment:
Could you wrap the `<spring:message>` and the `<select>` in a `<label>`? That would improve the accessibility for this! 😄